src/main/java/mops/businesslogic/DirectoryService.java

```diff
@@ -89,4 +89,19 @@ DirectoryPermissions updatePermission(Account account,
      */
     Directory getDirectory(long dirId) throws MopsException;
 
+    /**
+     * Get the total number of directories in a group.
+     *
+     * @param groupId group
+     * @return directory count
+     */
+    long getDirCount(long groupId) throws MopsException;
+
+    /**
+     * Get the total number of directories in all groups.
+     *
+     * @return directory count
+     */
+    long getDirCount() throws MopsException;
```

Comment:
This is a bit confusing, maybe rename one of them

Reply:
sure